core/domain/story_services.py

```diff
@@ -345,22 +345,35 @@ def _save_story(committer_id, story, commit_message, change_list):
     topic = topic_fetchers.get_topic_by_id(
         story.corresponding_topic_id, strict=False)
 
+
     story_is_published = False
+    story_is_present_in_topic = False
     if topic is not None:
```

Comment:
Maybe handle the `if topic is None` case and avoid having else case (this is the pattern we follow and it makes it more readable.)

Reply:
Done